expo/expo-cli
# [doctor] Add version validation for expo-modules-autolinking, @expo/config-plugins, @expo/config

# Why

Moving from SDK 44 -> 45 we found that many developers encountered issues when they depended on libraries that included old versions of `expo-modules-autolinking` or `@expo/config-plugins` (eg: https://github.com/expo/expo-cli/issues/4380).

# How

Use `npm explain` to find every version of each of the packages used in the project and verify that they all satisfy the version requirements from the versions endpoint.

# Test Plan

- Initialize a project and install `@stripe/stripe-react-native@0.6.0` (exact version)
- Run doctor, verify that you are warned about it
- Install expo-cli to your project dependencies
- Run doctor, verify that you are warned about `expo-modules-autolinking`
